src/features/client/pages/CourierRegisterLanding/index.jsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+import './index.css';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  ChakraProvider,
+  Heading,
+  Flex,
+  Image,
+  List,
+  ListIcon,
+  ListItem,
+  Stack,
+  Text,
+} from '@chakra-ui/react';
+import { CheckCircleIcon } from '@chakra-ui/icons';
+import ExtendedComponent from './ExtendedComponent';
+import { Link } from 'react-router-dom';
+
+function CourierRegisterLanding() {
+  return (
+    <React.Fragment>
+      <ChakraProvider>
```

Comment:
Chakra provider should only be used once. You need to create a layout component that injects both courier register landing and form.